packages/core/theme-module/index.js

```diff
@@ -32,14 +30,12 @@ module.exports = function DefaultThemeModule(moduleOptions) {
 
   this.options.dir = {
     ...this.options.dir,
-    ...{
-      layouts: '.nuxt/layouts',
-      assets: '.nuxt/assets',
-      pages: '.nuxt/pages'
-    }};
+    layouts: '.nuxt/layouts',
+    assets: '.nuxt/assets',
+    pages: '.nuxt/pages'
+  };
 
   this.extendBuild(config => {
-    delete config.resolve.alias['~'];
```

Comment:
i think this will break assets resolution

Reply:
Are you sure? 3 lines later it's overwritten anyway, so I think removing this line will not break anything. :)